appcli/cli_builder.py

```diff
@@ -270,7 +270,7 @@ def relaunch_if_required(ctx: click.Context):
             )
 
         for name, value in cli_context.additional_env_variables:
-            command.extend(shlex.split(f"--env {name}=\"{value}\""))
+            command.extend(shlex.split(f'--env {name}="{value}"'))
```

Comment:
Black formatter's handiwork. Still works as expected as f-strings work with single quotes too, now we just don't need the escaping